03-task-manager/starter/app.js

```diff
@@ -1 +1,38 @@
-console.log('Task Manager App')
+
+
+const express = require ('express')
+const app = express();
+const tasks = require('./routes/tasks')
+const connectDB = require('./db/connect')
+require('dotenv').config()
```

Comment:
Nitpick:
I'd recommend moving this require('dotenv').config(); to the top of the file. If you were to have a file that invokes a function which uses an ENV, then on import/require I believe the ENV will be undefined.